### ISSUE TITLE
Pathwright incorrectly alerts that survey was not filled

### ISSUE THREAD

**Issue by voscarmv** [microverseinc/microverse-issue-tracker#224]
When submitting the survey at https://microverse.pathwright.com/library/microverse-onboarding/86777/path/step/55995706/ the page shows me an alert slider from the right side of the page, mistakenly alerting me that I did not complete the survey.

The error happens after the page refreshes from clicking "complete step", and clears the survey answers. This seems to give the page the false impression that one did not complete the survey, once  "Next" is pressed.

**Comment by Keqi** [comment]
Hi @voscarmv,

I am extremely sorry for the very late response. Our development resources are very limited recently. We'll look into it and get back to you in time.

**Comment by gregthepeg4** [comment]
Hi @voscarmv I tried reproducing the error you described but it worked perfectly for me. Could you try and complete the survey again and make sure you submit the form before you click "complete step". Please record your screen while you do this so we can see what problem is occurring.

**Comment by voscarmv** [comment]
@Keqi no worries.

@gregthepeg4 I can't reproduce the error either... Now I'm not sure if it really happened how I remember, or if maybe I made a mistake while trying to submit the form.

What I do remember distinctly was that the error message stuck even after I tried filling the form several times. I don't know, it might have been just a fluke. It may be too difficult to reproduce to give it too much importance perhaps.

Anyway, sorry to bother you and thank you for your response.
